package.json

```diff
@@ -12,13 +12,14 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "turbo run build",
+    "build": "cd packages/sdk && yarn typechain && cd ../.. && turbo run build lint",
     "build:app": "cd packages/sdk && yarn typechain && turbo run build --filter=@stackly/sdk --filter=app",
     "build:landing": "cd packages/sdk && yarn typechain && turbo run build --filter=@stackly/sdk --filter=landing",
     "dev": "turbo run dev",
     "start": "turbo run start",
     "lint": "turbo run lint",
     "clean": "turbo run clean",
+    "clean:deep": "git clean -fdx",
```

Comment:
Why this git command? Show we add git commands to `package.json`?

Reply:
I left a brief explanation in the PR comment:
* Create a new `clean:depp` command useful to rebuild from scratch. It removes:
  * All `untracked` files
  * `-f` to force the deletion
  * `-d` to remove untracked directories
  * `-x` but only files and directories ignored by Git (present in `.gitignore`).
  
I'm not 100% sure about placing this here, but wanted to know what you guys feel about it. I think it's a useful command (that I didn't know about) and wanted to share it but I don't mind if we don't have it here. Feels more like a local command alias than a package command TBH.

Reply:
Yeah, totally, removing now